src/ui-tray-icon.c

```diff
@@ -166,9 +166,11 @@ pixbuf_array_free(GdkPixbuf **pixbufs)
 static GdkPixbuf **
 pixbuf_array_new(int size)
 {
-	GdkPixbuf *pixbufs[N_VOLUME_PIXBUFS];
+	GdkPixbuf **pixbufs;
 	gboolean system_theme;
 
+	pixbufs = g_malloc(sizeof(GdkPixbuf*) * N_VOLUME_PIXBUFS);
```

Comment:
Here I would use `g_new()`, I find it more readable. I would even use `g_new0()` to initialize the array to zero at the same time, it's always better than having junk values in memory, as long as performance doesn't matter.

    pixbufs = g_new0(GdkPixbuf *, N_VOLUME_PIXBUFS);